test/Utils.test.ts

```diff
@@ -57,10 +57,7 @@ suite('EditorConfig extension', () => {
 					insertSpaces: false,
 					tabSize: 4
 				},
-				expected: {
-					insertSpaces: false,
-					tabSize: 5
-				}
+				expected: { }
```

Comment:
Just trying to understand this. So if the `indent_size = 5` but the workspace or user settins are `insertSpaces: false` and `tabSize: 4` then we expect the resulting config to be empty? Which essentially just defers to the same `insertSpaces: false` and `tabSize: 4` that was in the workspace or user settings anyway, right?

It's been a while since I've delved into this tests, but it seems the "defaults" are less and less relevant after these changes, which might be a good thing.